privacy-policy.md

```diff
@@ -0,0 +1,131 @@
+
+- Cloud hosting providers (e.g., AWS, Google Cloud)
+- Email marketing platforms (e.g., Mailchimp)
+- Analytics tools (e.g., Google Analytics)
+
+We ensure all third-party providers comply with GDPR and applicable regional privacy laws through data processing agreements. Sponsors may receive aggregated data (e.g., event participation metrics) without identifying individual users.
+
+## 6. International Data Transfers
+
+As PAO operates across East and Southeast Asia, your data may be transferred to countries outside the European Economic Area (EEA). To safeguard your data:
+
+- We use Standard Contractual Clauses (SCCs) for data transfers.
+- We ensure equivalent levels of data protection in compliance with GDPR and regional privacy regulations.
+
+## 7. Data Security
+
+We take appropriate technical and organizational measures to secure your personal data, including:
+
+- Data encryption during transmission and storage.
+- Regular security audits.
+- Access controls to restrict data access to authorized personnel only.
+
+## 8. Your Rights
+
+Under GDPR and applicable privacy laws, you have the following rights:
+
+- **Access:** Request a copy of your personal data.
+- **Rectification:** Correct inaccurate or incomplete information.
+- **Erasure:** Request deletion of your data (subject to legal obligations).
+- **Restriction:** Limit how your data is processed.
+- **Portability:** Receive your data in a structured, machine-readable format.
+- **Objection:** Object to data processing based on legitimate interest.
+- **Withdrawal of Consent:** Withdraw your consent at any time for specific processing activities.
+
+To exercise your rights, email us at [privacy@pythonasia.org](mailto:privacy@pythonasia.org).
+
+## 9. Data Retention
+
+We retain personal data only as long as necessary to fulfill the purposes outlined in this policy or comply with legal obligations. Regular reviews ensure that data is not retained longer than required.
+
+## 10. Cookies Policy
+
+We use cookies to:
+
+- Enable essential website functionality.
+- Analyze website traffic and user behavior.
+- Enhance user experience.
+
+You can manage cookie preferences through your browser settings or our cookie management tool (if available). Non-essential cookies require your explicit consent.
+
+## 11. Third-Party Links
+
+Our website may contain links to external websites. We are not responsible for the privacy practices or content of these websites. We encourage you to review their privacy policies.
+
+## 12. Changes to This Privacy Policy
+
+We may update this Privacy Policy to reflect changes in legal requirements, operations, or sponsorship programs. Significant updates will be communicated via email or website notifications. Please review this policy periodically.
+
+## 13. Contact Us
+
+For questions, concerns, or to exercise your rights, please contact us:
+
+**Email:** [privacy@pythonasia.org](mailto:privacy@pythonasia.org)  
+**Mailing Address:** [Insert Address]
```

Comment:
Tokyo, Japan our office address.
"110-0015 Tokyo-to, Taito-ku, Higashi-Ueno 1-11-1, GOSHO Kasugadori Building, 9th Floor."

Reply:
Yes, I think this is fine because we're using our "mailing address" here.